### PR TITLE
Header change for lists

### PR DESCRIPTION
**Pull request by frankTheTank72**

### CODE REVIEW

File: scan/templates/blocks/detail.html [hunk 1]
```diff
@@ -7,10 +7,13 @@
 
 {% block content %}
 
-  <h5 class="p-2">Block #{{ blk.id }}</h5>
+
```
Comment (ohager) [inline scan/templates/blocks/detail.html:10]: you are omitting the blocks id? I mean we always use the height and not the id....so what

File: static/css/style.css [hunk 2]
```diff
@@ -28,6 +28,9 @@ body, html {
   background-color: #0099ff !important;
 }
 
+.card-header:first-child {
+    border-radius: 0.8em 0.8em 0 0;
```
Comment (ohager) [inline static/css/style.css:32]: is that border-radius is used all over? or just your personal style opinion? It's more important to be consistent than just opiniated guesses

**Review by ohager — COMMENTED** [review]
I left some comments

File: scan/templates/blocks/detail.html (hunk 1 above)
Comment (frankTheTank72) [reply scan/templates/blocks/detail.html:10]: good catch was in the wrong template - weill revert

File: static/css/style.css (hunk 2 above)
Comment (frankTheTank72) [reply static/css/style.css:32]: yes is no in line with the boarder of the card

Comment (frankTheTank72) [reply static/css/style.css:32]: Did a new commit in this PR

File: scan/templates/blocks/detail.html (hunk 1 above)
Comment (frankTheTank72) [reply scan/templates/blocks/detail.html:10]: Did a new commit for this file

**Review by ohager — APPROVED** [review]
lgtm

**Pull request merged**